client/components/calypso-i18n-provider/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useState, useEffect } from 'react';
 import type { I18N } from 'i18n-calypso';
 import type { FunctionComponent } from 'react';
 
-const CalypsoI18nProvider: FunctionComponent< { i18n: I18N } > = ( {
+const CalypsoI18nProvider: FunctionComponent< { i18n?: I18N } > = ( {
```

Comment:
How is this component expected to work with an undefined `i18n` property? This seems problematic. Perhaps we should just not use the provider at all in that case?